lib/elixir/lib/base.ex

```diff
@@ -124,10 +122,7 @@ defmodule Base do
   end
 
   defp shift(alphabet) do
-    alphabet
-    |> length()
-    |> :math.log2()
-    |> round()
+    alphabet |> length() |> :math.log2() |> round()
```

Comment:
Pipelines read better as multi lines.

Reply:
Pipelines should *always* be multi-line, just 'always'.